jerry-ext/include/jerryscript-ext/section.impl.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
I didn't intend for it to be related to modules. It contains macros that help with linker sections, whatever purpose those linker sections may serve.